presto-native-execution/src/test/java/com/facebook/presto/spark/PrestoSparkNativeQueryRunnerUtils.java

```diff
@@ -90,7 +90,7 @@ public static Map<String, String> getNativeExecutionSessionConfigs()
                 .put("native-execution-enabled", "true")
                 .put("spark.initial-partition-count", "1")
                 .put("register-test-functions", "true")
-                .put("native-execution-program-arguments", "--logtostderr=1 --minloglevel=3")
+                .put("native-execution-program-arguments", "--logtostderr=1 --minloglevel=0")
```

Comment:
Is this necessary? This may produce too much logs on the CI machine.

Reply:
This is for testing and will remove when land.